class12/dlqkfzlr/study/app/src/main/java/m/woong/architecturestudy/ui/adapter/MovieAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package m.woong.architecturestudy.ui.adapter
+
+import android.text.Html
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import m.woong.architecturestudy.R
+import m.woong.architecturestudy.ui.item.MovieItem
+
+class MovieAdapter(private val movieList: ArrayList<MovieItem>) :
+    RecyclerView.Adapter<MovieAdapter.MovieViewHolder>() {
+
+    class MovieViewHolder(v: View) : RecyclerView.ViewHolder(v) {
+        val imageImageView: ImageView
```

Comment:
```suggestion
        val imageImageView: ImageView = v.movie_image 
```
이런식으로 바로 선언하고 init은 빼도 괜찮지 않을까요?

Reply:
제안사항 반영했습니다 :)
(b68d69)